server/tidb_test.go

```diff
@@ -1906,10 +1906,21 @@ func (c *resourceTagChecker) checkExist(t *testing.T, digest stmtstats.BinaryDig
 
 	c.Lock()
 	defer c.Unlock()
-	_, ok := c.sqlDigests[digest]
+	_, ok := c.sqlDigest2Reqs[digest]
 	require.True(t, ok, sqlStr)
 }
 
+func (c *resourceTagChecker) checkReqExist(t *testing.T, digest stmtstats.BinaryDigest, sqlStr string, reqs ...tikvrpc.CmdType) {
+	c.Lock()
+	defer c.Unlock()
+	reqMap, ok := c.sqlDigest2Reqs[digest]
+	require.True(t, ok, sqlStr)
+	for _, req := range reqs {
+		_, ok := reqMap[req]
+		require.True(t, ok, sqlStr+"--"+req.String())
```

Comment:
Only from this PR I can't determine the corresponding source of this format (`sqlStr+"--"+req.String()`), can we extract this string concatenation into a function?